include/config_directives.h

```diff
@@ -45,6 +45,7 @@ CFGFUN(for_window, const char *command);
 CFGFUN(gaps, const char *workspace, const char *type, const long value);
 CFGFUN(smart_borders, const char *enable);
 CFGFUN(smart_gaps, const char *enable);
+CFGFUN(hide_combined_titles, const char *enable);
```

Comment:
I don't like this name too much. Can we change it to `force_tabbed_stacked_titlebar yes|no` with `yes` being the default?